README.md

```diff
@@ -682,24 +689,18 @@ doSomethingWithClosure() { response: NSURLResponse in
 [1, 2, 3].flatMap { String($0) }
 ```
 
-* **3.8.2** When listing parameters, if using a capture list and/or specifying a (non-void) return type, wrap the parameter list in parentheses. Otherwise, the parentheses can be avoided.
+* **3.8.2** When listing parameters in a closure, wrap the parameter list in parentheses.
 
 ```swift
-// parentheses due to capture list
-doSomethingWithClosure() { [weak self] (response: NSURLResponse) in
+doSomethingWithClosure() { [weak self] (response) in
     self?.handleResponse(response)
 }
-
-// parentheses due to return type
-doSomethingWithClosure() { (response: NSURLResponse) -> String in
-    return String(response)
-}
 ```
 
 * **3.8.3** If specifying a closure as a type, you don’t need to wrap it in parentheses unless it is required (e.g. if the type is optional or the closure is within another closure). Always wrap the arguments in the closure in a set of parentheses - use `()` to indicate no arguments and use `Void` to indicate that nothing is returned.
 
 ```swift
-let completionBlock: (success: Bool) -> Void = {
+let completionBlock: (Bool) -> Void = { success in
```

Comment:
It is. The rule is talking about wrapping the entire declaration in parentheses, e.g. `((Bool -> Void)` instead of `(Bool) -> Void`.